### DISCUSSION TITLE
[Esame 18 ottobre 2024] Esercizio 2

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#606] [Esami M-Z]
![2](https://github.com/user-attachments/assets/c617089f-f021-4fdb-89e4-77f23f60807f)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/b4f6f9a4-e218-4f30-9dad-749d9d11be8a)

**Comment by niccolo-fato** [comment]
<img width="1280" height="1042" alt="image" src="https://github.com/user-attachments/assets/981dc79d-9d25-4426-811e-8596f3f707c6" />
